### DISCUSSION TITLE
Regarding Qualcomm X-Elite and X-Plus device availability in the openssl daily test runs in Github

### DISCUSSION THREAD

**Discussion by quicmmarimut** [openssl/openssl#26034] [Q&A] [answered]
Hi Team,
   This question is regarding the Qualcomm X-Elite and X-Plus device availability in the openssl daily test runs in Github. Currently I see the tests are running on a daily basis and also for every commit that we are making. 

 Does this daily runs are running on only x64 based intel machines ? Or do we have any Qualcomm (X Elite or X Plus) based devices also ? Can you please clarify on this ?

Thanks & Regards,
Mallikarjunan

**Comment by t8m** [comment, marked as answer]
There are no Qualcomm X-Elite and X-Plus devices in the CI. The CI tests are mostly done on Linux, Windows and MacOS github CI runners provided by Github.

**Reply by quicmmarimut** [reply]
Thank you for your response !
